src/openzeppelin/utils/structs/enumerableset.cairo

```diff
@@ -0,0 +1,141 @@
+# SPDX-License-Identifier: MIT
+# OpenZeppelin Cairo Contracts v0.1.0 (utils/structs/enumerableset.cairo)
+%lang starknet
+
+from starkware.cairo.common.cairo_builtins import HashBuiltin
+from starkware.cairo.common.alloc import alloc
+from starkware.cairo.common.bool import TRUE, FALSE
+
+@storage_var
+func EnumerableSet_values(set_id : felt, set_index : felt) -> (value : felt):
+end
+
+@storage_var
+func EnumerableSet_sizes(set_id : felt) -> (size : felt):
```

Comment:
yea, originally i tried to do it based on pointers and `alloc` but that doesnt work in native cairo because of the immutable memory. this was the only way i could think of. in my mind having a constant for the `setId` would be the same as defining the variable in solidity